DeepProjection/tests/test_pdbGenerator.py

```diff
@@ -13,4 +13,182 @@ def test_getting_pdb(empty_pdbgenerator):
     pdbID, pdbFile = empty_pdbgenerator.get_random_pdb()
     assert empty_pdbgenerator.pdbID is not None
 
+def test_download_pdb_in_range(empty_pdbgenerator):
+    '''download_pdb_in_range() can be run using an "empty" PDBGenerator'''
+    '''download_pdb_in_range() returns True when it encounters no input errors; does not mean it successfully downloaded a file'''
 
+    """
+    Default values
+        1. If upper == None, then range is from [lower, 9ZZZ].
+        2. If dir == None, then download PDB files to current working directory.
+    """
+    # There is no PDB with an ID of '9zzz', so it will be used in default value test.
```

Comment:
9zzz may appear one day and this would fail. I think a better check would be a pdb starting with zero, i.e. 0aaa